config.json

```diff
@@ -211,6 +216,18 @@
           "records"
         ],
         "status": "beta"
+      },
+      {
+        "slug": "go",
+        "name": "Go",
+        "uuid": "33c34af3-26fa-4e18-8bb3-8af688069c9e",
+        "concepts": [
+          "result"
+        ],
+        "prerequisites": [
+          "maybe"
```

Comment:
Just to be sure: this does not yet list all prerequisites mentioned in the design document: https://github.com/exercism/elm/pull/438/files#diff-df035168d36af2aa6b326789fa41efffae7f77b2c07a54dd0c26502cbafacd8dR24

The website will ignore any prerequisites that are not taught by any concept exercise (or that don't exist), so you _could_ add it to this list I think. But you can also do that later of course.

Reply:
Good spot, I've added these to the config.